### PR TITLE
[prometheus-mysql-exporter] Add relabelings to serviceMonitor

### PR DESCRIPTION
**Pull request by Mahanotrahul**
#### What this PR does / why we need it:
PR to add relabelings in servicemonitor
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [ ] Chart Version bumped
- [ ] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-mysql-exporter/values.yaml [hunk 1]
```diff
@@ -109,6 +112,7 @@ collectors: {}
 mysql:
   db: ""
   host: "localhost"
+  # host: &host "localhost"
```
Comment (monotek) [inline charts/prometheus-mysql-exporter/values.yaml:115]: What is that comment for?

**Review by monotek — CHANGES_REQUESTED** [review]

**Comment by monotek** [comment]
Please sign your commits to get the dco check pass.

**Review by Juanchimienti — DISMISSED** [review]
looks good

File: charts/prometheus-mysql-exporter/values.yaml (hunk 1 above)
Comment (Mahanotrahul) [reply charts/prometheus-mysql-exporter/values.yaml:115]: Usually prometheus will have the ip of the mysql exporter in the instance label. But if we want to have the actual host name of the mysql in the instance label, we can achieve it using anchors. And so I've added it as a comment for anyone to make use of it.

Comment (monotek) [reply charts/prometheus-mysql-exporter/values.yaml:115]: Can we remove this please?
Anchors are kind of discuraged from using with helm. See: https://helm.sh/docs/chart_template_guide/yaml_techniques/#yaml-anchors

**Review by monotek — CHANGES_REQUESTED** [review]

**Pull request closed without merging**